<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>summaDeviceConfig</class>
 <widget class="QDialog" name="summaDeviceConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>307</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="deviceTable">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Device</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Path</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="newButton">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Edit Device</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0">
      <item>
       <widget class="QComboBox" name="deviceTypeCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="devicePathEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deviceSaveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset theme="document-save">
          <normaloff/>
         </iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>summaDeviceConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>summaDeviceConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newButton</sender>
   <signal>clicked()</signal>
   <receiver>summaDeviceConfig</receiver>
   <slot>newDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>summaDeviceConfig</receiver>
   <slot>deleteDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deviceSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>summaDeviceConfig</receiver>
   <slot>saveDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newDevice()</slot>
  <slot>deleteDevice()</slot>
  <slot>deviceClicked(int,int)</slot>
  <slot>saveDevice()</slot>
 </slots>
</ui>
